src/datumaro/plugins/transforms.py

```diff
@@ -1967,12 +1981,22 @@ def transform_item(self, item):
                 "Item %s: TableRow info is required for this " "transform" % (item.id,)
             )
 
+        sep_token = self._sep_token
         refined_media = self.refine_tabular_media(item) if item.media.has_data else None
-        annotation_values = {
-            key: refined_media.data[key] for key in item.annotations[0].values.keys()
-        }  # only for tabular
-        refined_annotations = [
-            ann.wrap(values=annotation_values) if isinstance(ann, Tabular) else ann
-            for ann in item.annotations
-        ]
+        refined_annotations = []
+        for ann in item.annotations:
+            if isinstance(ann, Tabular):
+                annotation_values = {
+                    key: refined_media.data[key] for key in item.annotations[0].values.keys()
```

Comment:
What's the reason that we do not care `item.annotations[i].values` for `i=1,...` not just `i=0`?

Reply:
If a dataset item contains a tabular annotation type, it will contain only one annotation of tabular for each item. This item will not contain any other annotations. I think this part is a bit hard-coded, so if there is another way to check the annotation type, please let me know.